File: modul4/C/PRAK402_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int batas_maksimal;
    
    while (1) {
    printf("");
    scanf("%d", &batas_maksimal);

    for (int i = 1; i <= batas_maksimal; i += 2) {
        printf("%d ", i);
    }

    printf("\n");
    if (batas_maksimal % 2 == 0) {
        for (int i = batas_maksimal; i >= 2; i -= 2) {
            printf("%d ", i);
        }
    } else {
        for (int i = batas_maksimal - 1; i >= 2; i -= 2) {
            printf("%d ", i);
        }
    }

    printf("\n");

    }

    return 0;
}

File: modul4/Python/PRAK402_2310817210017_MUHAMMADNURHUDABAIHAQI.py
while True:
    batas_maksimal = int(input(""))

    if batas_maksimal == 0:
        break  

    for i in range(1, batas_maksimal + 1, 2):
        print(i, end=" ")

    print()

    if batas_maksimal % 2 == 0:
        for i in range(batas_maksimal, 0, -2):
            print(i, end=" ")
    else:
        for i in range(batas_maksimal - 1, 0, -2):
            print(i, end=" ")

    print()
